### PR TITLE
docs: improve grammar in gh templates

### PR DESCRIPTION
**Pull request by skanda890**
## Previous Behavior
.md files have mistakes.
## New Behavior
Now .md files don't have mistakes.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/convergence_epic.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 name: (internal) Component implementation epic
 about: (team member use only) Epic issue tracking implementation of a particular component
----
 
+---
```
Comment (JustSlone) [inline .github/ISSUE_TEMPLATE/convergence_epic.md:5]: This change doesn't seem to have an effect. What is it fixing, and is it necessary?

Comment (Hotell) [reply .github/ISSUE_TEMPLATE/convergence_epic.md:5]: this is breaking frontmatter as well

**Comment by skanda890** [comment]
Done.

**Review by Hotell — APPROVED** [review]
LGTM

**Comment by skanda890** [comment]
> LGTM

Meaning?

**Comment by skanda890** [comment]
Also, check #32131

**Comment by JustSlone** [comment]
> > LGTM
> 
> Meaning?

This is shorthand for "Looks Good To Me" it's used to express sign off on the PR

**Review by JustSlone — APPROVED** [review]

**Review by khmakoto — APPROVED** [review]

**Pull request merged**